### ISSUE TITLE
checkbox list items continuation with mkdocs flavour are formated wrongly

### ISSUE THREAD

**Issue by jankatins** [rvben/rumdl#529]
### rumdl version

0.1.47

### Installation method

cargo install

### Bug description

(follow up of https://github.com/rvben/rumdl/issues/522#issuecomment-4048545155)

If I have a checkbox style list, I get this:

<img width="149" height="173" alt="Image" src="https://github.com/user-attachments/assets/e8cd18b1-790e-465c-85ec-543471cb5aa2" />

And I cannot change this: if I manually add 4 spaces in the continuation lines, it will reformat it again to zero :-( If I remove the checkmark, the 4 spaces are applied.

### Steps to reproduce

Add a list with checkmarks:

```markdown
- [ ] whatever long line which goes on and on and on and on and on and on and on and on and on and on.
   A continuation line which should according to the mkdown flavour get formatted with 4 spaces.
```

format it with the below config

### Expected behavior

```markdown
- [ ] whatever long line which goes on and on and on and on and on and on and on and on and on and on.
     A continuation line which should according to the mkdown flavour get formatted with 4 spaces.
```

(4 spaces in the continuation line)

### Actual behavior

```markdown
- [ ] whatever long line which goes on and on and on and on and on and on and on and on and on and on.
A continuation line which should according to the mkdown flavour get formatted with 4 spaces.
```

Zero spaces in the continuation lines

### Configuration

```toml
[tool.rumdl]
# We need mkdocs for backstage (can be overridden by per file rules)
flavor = "mkdocs"
exclude = [
 # ...
]
respect-gitignore = true

# Set global line length (used by MD013 and other line-length rules)
line_length = 120

[tool.rumdl.MD013]
# https://rumdl.dev/md013/
# Reflow at semantic line breaks: https://sembr.org/
reflow = true
reflow-mode = "semantic-line-breaks"
code-blocks = false  # Don't check code blocks
tables = false  # Don't check tables
headings = false  # Check headings
paragraphs = true  # Check paragraph/regular text
strict = false  # Exceptions for URLs, etc.
length-mode = "visual"  # How to count line length: "visual", "chars", or "bytes" (default: "visual")
abbreviations = ["Assn", "Univ"]  # Add custom abbreviations for sentence-per-line mode
require-sentence-capital = true  # Require uppercase after periods for sentence detection

[tool.rumdl.MD004]
style = "dash"

[tool.rumdl.MD029]
# https://github.com/rvben/rumdl/blob/main/docs/md029.md
# ordered lists: either all 1. or properly ordered.
style = "one-or-ordered"
```

### Operating system

Linux

**Comment by rvben** [comment]
Fixed in v0.1.48. Checkbox list item continuation indent now uses the base marker width for MkDocs flavor, producing correct indentation. Thanks for the follow-up, @jankatins!
